hazelcast/src/main/resources/hazelcast-full-example.yaml

```diff
@@ -69,8 +69,9 @@ hazelcast:
           secretKeyFactoryAlgorithm: PBKDF2WithHmacSHA1
   #
   # Specifies the name and password for a cluster group you create.
-  # Cluster groups allow you to create separate sub-clusters within your Hazelcast cluster to
-  # create a simple security. They are also referenced in the WAN Replication configuration.
+  # Cluster groups allow you to create separate sub-clusters s with different names.
```

Comment:
Perhaps a typo with the extra 's' in "...sub-clusters s with..."

Reply:
Nice catch ;). Thanks. I will fix it.